charts/powerdns/Chart.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v2
+appVersion: "latest"
+description: PowerDNS is a DNS server, written in C++ and licensed under the GPL. It runs on most Unix derivatives. PowerDNS features a large number of different backends ranging from simple BIND style zonefiles to relational databases and load balancing/failover algorithms. A DNS recursor is provided as a separate program.
+name: powerdns
+version: 0.3.0
+home: https://www.powerdns.com/
+sources:
+  - http://www.github.com/PowerDNS/
+icon: https://www.powerdns.com/img/pDNS_logo_positive_claim_down_screen_xsmall.png
+maintainers:
+  - name: Ryan Holt
```

Comment:
From what I recall, this needs to be a github username, so `carpenike`

Reply:
aha, ok. trying again. ty.